pages.fr/common/pip.md

```diff
@@ -3,30 +3,34 @@
 > Gestionnaire des paquets pour Python.
 > Plus d'informations : <https://pip.pypa.io>.
 
-- Installe un paquet :
+- Installe un paquet:
```

Comment:
Putting a space before every colon is France writing style, not a typo.
```suggestion
- Installe un paquet :
```

Reply:
@blueskyson but it has to be a non-breaking space.
```suggestion
- Installe un paquet :
```